DemonsDiseases/Barya.xml

```diff
@@ -0,0 +1,51 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="Barya" xml:lang="en" type="auth">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Bāryā</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="ES"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A cause of disease,  frequently target of <term key="Magic">magic prayers</term>; see also <persName ref="ETH1219baryas"></persName>.</p>
```

Comment:
not <p><term key="demon">Demonic entities</term>? there is a social group, the 'slaves' and there is a deamon, no? @DenisNosnitsin1970 @karljonaskarlsson

Reply:
I think Nafisa's observation is correct: ETH1219baryas seems to refer to an ethnic group and not to a type of demons. It is used for a group of military units in Amda Seyon's chronicle, for example

Reply:
ETH1219baryas  is a social group, the ID is not used for the military unit, see now also https://github.com/BetaMasaheft/Persons/pull/914